shell/platform/common/cpp/text_input_model.h

```diff
@@ -47,6 +53,19 @@ class TextInputModel {
   // Returns true if any deletion actually occurred.
   bool Delete();
 
+  // Deletes text near the cursor.
+  //
+  // A section is made starting at @offset code points past the cursor (negative
+  // values go before the cursor). @count code points are removed. The selection
+  // may go outside the bounds of the text and will result in only the part
+  // selection that covers the available text being deleted. The existing
+  // selection is ignored and removed after this operation. Calling this with
+  // (-1, 1) is equivalent to Backspace(). Calling this with (0, 1) is
+  // equivalent to Delete().
```

Comment:
These two examples aren't actually correct since those methods don't ignore selection, so I would remove them. I think the new explanation before the examples is clear enough without them.